src/app/utilities/AnnotateUtilities.ts

```diff
@@ -208,14 +209,33 @@ const GenerateAnnotationFormFieldProperties = async (jsonPath: string, superClas
 
     /**
      * Function to check for a default value for a term
-     * 
+     * @param key The term key to check
      */
     const CheckForTermDefaultValue = (key: string) => {
         if (toLower(key).includes('date')) {
             return format(new Date, 'yyyy-MM-dd');
         }
     };
 
+    /**
+     * Function to depict the values to be set for the class to be annotated
+     * @param fieldName The field name of the class (parsed from JSON path)
+     * @param classValues Possible existing class values from the super class
+     * @param isArray Boolean indicating if the class represents an array
+     * @param localClassValues Possible existing local class values of an annotation
+     */
+    const DepictClassValues = (fieldName: string, classValues: Dict | Dict[] | undefined, isArray?: boolean, localClassValues?: Dict[]) => {
+        if (classValues) {
+            return classValues;
+        } else if (!isEmpty(localClassValues)) {
+            return localClassValues;
+        } else if (motivation !== 'oa:editing' && isArray) {
```

Comment:
should this include 'oa:adding' ?

Reply:
Not this time. Prefill values is only allowed for the 'ods:adding' motivation.
Actually changed this check to be: motivation === 'ods:adding', to also exclude the other motivations.